header-generator/translate.go

```diff
@@ -185,15 +185,15 @@ func (xl8r *ScalarConverter) GoToC() string {
 }
 
 var (
-	Int8Translator          *ScalarConverter = &ScalarConverter{"int8_t", "C.schar", "int8"}
-	Int16Translator                          = &ScalarConverter{"int16_t", "C.short", "int16"}
-	Int32Translator                          = &ScalarConverter{"int32_t", "C.int", "int32"}
-	Int64Translator                          = &ScalarConverter{"int64_t", "C.longlong", "int64"}
-	Uint8Translator                          = &ScalarConverter{"uint8_t", "C.uchar", "byte"}
-	Uint16Translator                         = &ScalarConverter{"uint16_t", "C.ushort", "uint16"}
-	Uint32Translator                         = &ScalarConverter{"uint32_t", "C.uint", "uint32"}
-	Uint64Translator                         = &ScalarConverter{"uint64_t", "C.ulonglong", "uint64"}
-	SizeTranslator                           = &ScalarConverter{"size_t", "C.ulong", "uint64"}
```

Comment:
Is size_t actually a uint64 on your platform? (a unsigned long long). This line is breaking on my intel mac because size_t is actually
`/Library/Developer/CommandlineTools/SDKs/MacOSX12.3.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/stddef.h:49:typedef __typeof__(sizeof(int)) size_t;`

Curious what platform you are one, and if this will need to be made more dynamic / autodetected.

Reply:
I am using Ubuntu 22.04 on 11th gen Intel CPU. Size of size_t on my machine is 64 bits. 
I think that there is no need for dynamic detection. Can you test commit "Fix cgo data type size_t"?